src/components/SelectInput/SelectInput.module.scss

```diff
@@ -0,0 +1 @@
+/* Placeholder SCSS file for future updates */
```

Comment:
There should probably only be one scss file in a dir. What is the reason for having two?